baseline/OSPS-AC.yaml

```diff
@@ -286,6 +297,11 @@ controls:
           - reference-id: AC-20(1)
           - reference-id: CM-5
           - reference-id: CM-7
+      - reference-id: UKSSCOP
+        entries:
+          - reference-id: Claim 2.1.1
+          - reference-id: Claim 2.1.3   
```

Comment:
I'm not sure that this item is intended to talk about project contributors' permissions _on_ the build pipeline.  (From the recommendation: "limiting the permissions granted to steps within the CI/CD pipelines")

2.1.3 seems like it is closer to a build-system equivalent of AC-02.01, though I read it as simply saying "everyone gets their own login on e.g. Jenkins -- don't use a shared account".